snapd-glib/snapd-client-sync.c

```diff
@@ -512,6 +512,10 @@ G_GNUC_END_IGNORE_DEPRECATIONS
  * If @flags contains %SNAPD_GET_SNAPS_FLAGS_INCLUDE_INACTIVE then also return snaps
  * with status %SNAPD_SNAP_STATUS_INSTALLED.
  *
+ * If @flags contains %SNAPD_GET_SNAPS_FLAGS_REFRESH_INHIBITED, then it will return
```

Comment:
Document that only one of SNAPD_GET_SNAPS_FLAGS_INCLUDE_INACTIVE or SNAPD_GET_SNAPS_FLAGS_REFRESH_INHIBITED can be set.

Reply:
Done.

Reply:
Mmm... but should I add a GError parameter, and set it if both flags are set?

Reply:
I think an assert would be more appropriate - treat it as a programmer error to set both flags.

Reply:
You are right. Done.